docs/usage/graphql.rst

```diff
@@ -0,0 +1,102 @@
+Graphql
+#######
+
+Litestar does not support ``graphql`` out of the box. This can be done throught third party packages. 
+
+.. note::
+    Currently the only library that supports Litestar is `Strawberry <https://strawberry.rocks/docs/integrations/litestar#litestar>`_.
+    Check the official `Graphql <https://graphql.org/community/tools-and-libraries/?tags=python>`_  documentation to learn more about other graphql libraries.
+
+
+Installation
+============
+
+To get started simply install it with:
+
+
+.. code-block:: shell
+
+    pip install litestar[graphql]
```

Comment:
I don't think this is something that exists.

Reply:
I am not in favor adding detailed examples in the first place. I would rather prefer a link to https://strawberry.rocks/docs/integrations/litestar. I will let others weigh in on if this.